### PR TITLE
Add test for build.log missing file parsing

### PR DESCRIPTION
**Pull request by matthewrsj**
Signed-off-by: Matthew Johnson <matthew.johnson@intel.com>

### CODE REVIEW

File: tests/test_build.py [hunk 1]
```diff
@@ -314,6 +314,39 @@ def mock_util_call(cmd):
...
+                  'errors here\n'
+        m_open = mock_open(read_data=content)
+
+        with patch(open_name, m_open, create=True):
+            build.parse_build_results('testname', 0, fm)
+
+        build.util.call = call_backup
+
+        self.assertEqual(fm.files, ['/usr/testdir/file',
+                                    '/usr/testdir/file1',
```
Comment (bryteise) [inline tests/test_build.py]: The alignment of these looks funky (same for the escapes above actually).

**Comment by bryteise** [comment]
Looks good minus the alignment looking funny in github.

File: tests/test_build.py (hunk 1 above)
Comment (matthewrsj) [reply tests/test_build.py]: for this, yeah it's funky. for the escapes I was just trying to make them line up...

**Comment by matthewrsj** [comment]
Fixed alignment issues.

**Comment by bryteise** [comment]
+1

**Pull request merged**